example-custom/src/main/res/values-v26/colors.xml

```diff
@@ -1,4 +1,4 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
-    <color name="ic_launcher_background">#1c7917</color>
+    <color name="ic_launcher_background">#1CCCA5</color>
```

Comment:
I don't really mind about it. I'm just curious. Why spend any time changing this?